en/administration/centreon-ha/installation-4-nodes.md

```diff
@@ -0,0 +1,1019 @@
+    ocf:heartbeat:IPaddr2 \
+    ip="@VIP_IPADDR@" \
+    nic="@VIP_IFNAME@" \
+    cidr_netmask="@VIP_CIDR_NETMASK@" \
+    broadcast="@VIP_BROADCAST_IPADDR@" \
+    flush_routes="true" \
+    meta target-role="stopped" \
+    op start interval="0s" timeout="20s" \
+    stop interval="0s" timeout="20s" \
+    monitor interval="10s" timeout="20s" \
+    --group centreon
+```
+
+##### Httpd service
+
+```bash
+pcs resource create http \
+    systemd:httpd24-httpd \
+    meta target-role="stopped" \
+    op start interval="0s" timeout="40s" \
+    stop interval="0s" timeout="40s" \
+    monitor interval="5s" timeout="20s" \
+    --group centreon \
+    --force
+```
+
+##### Gorgone service
+
+```bash
+pcs resource create gorgone \
+    systemd:gorgoned \
+    meta target-role="stopped" \
+    op start interval="0s" timeout="90s" \
+    stop interval="0s" timeout="90s" \
+    monitor interval="5s" timeout="20s" \
+    --group centreon
+```
+
+##### centreon-central-sync service
+
+This service only exists in the context of *Centreon HA*. It provides real time synchronization for configuration files, images, etc.
+
+```bash
+pcs resource create centreon_central_sync \
+    systemd:centreon-central-sync \
+    meta target-role="stopped" \
+    op start interval="0s" timeout="90s" \
+    stop interval="0s" timeout="90s" \
+    monitor interval="5s" timeout="20s" \
+    --group centreon
+```
+
+##### SQL Broker
+
+```bash
+pcs resource create cbd_central_broker \
+    systemd:cbd-sql \
+    meta target-role="stopped" \
+    op start interval="0s" timeout="90s" \
+    stop interval="0s" timeout="90s" \
+    monitor interval="5s" timeout="30s" \
+    --group centreon
+```
+
+##### Centengine service
+
+```bash
+pcs resource create centengine \
+    systemd:centengine \
+    meta multiple-active="stop_start" target-role="stopped" \
+    op start interval="0s" timeout="90s" stop interval="0s" timeout="90s" \
+    monitor interval="5s" timeout="30s" \
+    --group centreon
+```
+
+##### Centreontrapd service
+
+```bash
+pcs resource create centreontrapd \
+    systemd:centreontrapd \
+    meta target-role="stopped" \
+    op start interval="0s" timeout="30s" \
+    stop interval="0s" timeout="30s" \
+    monitor interval="5s" timeout="20s" \
+    --group centreon
+```
+
+##### Snmptrapd service
+
+```bash
+pcs resource create snmptrapd \
+    systemd:snmptrapd \
+    meta target-role="stopped" \
+    op start interval="0s" timeout="30s" \
+    stop interval="0s" timeout="30s" \
+    monitor interval="5s" timeout="20s" \
+    --group centreon
+```
+
+#### Resource constraints
+
+When using the 4 nodes architecture, you must define some specific Constraints to specify where Resources could run. 
+
+In order to glue the Primary Database role with the Virtual IP, define a mutual Constraint:
+
+```bash
+pcs constraint colocation add "centreon" with master "ms_mysql-master"
+pcs constraint colocation add master "ms_mysql-master" with "centreon"
+```
```

Comment:
you've glued ms_mysql-master with centreon and not the vip, we don't want to link mysql resource with centreon resources when using a 4 nodes cluster